### ISSUE TITLE
feat(COMPONENT): mat-tab responsive

### ISSUE THREAD

**Issue by zensasian85** [angular/components#23495]
#### Feature Description

The current solution for the mat-tab responsive with the arrows are not quite functional, because you have to click on the arrows multiple times to get to or see the next tab, so it's quite annoying.
I've made a solution, where the "mat-tab-label-container" is scrollable and hide the arrows, but the scrollable solution breaks the 
click function on the tabs (had to click multiple times on the tab, before it got triggered).

My suggestion is that you should refactor the responsive solution.
I have 2 proposals described in "Use Case".

#### Use Case

**1: Scrollable**
Make the tab area scrollable without breaking the click function on tabs. Remove the arrows and replace them with a nice faded gradient, so the overflow looks smooth.

**2: Snap to next tab, when click on arrows**
Instead of you have to click multiple times on the arrows to get to next tab, you should only click once and it snaps to the next tab, so you can click on the tab.

One of these 2 solutions would be much more compliant both for desktop and responsive.

NB! There's a bug, when adding attribute "mat-stretch-tabs". The label text gets "eaten", when the text is a bit long. I had to overwrite css with "flex-basis: auto !important" to fix this issue

**Comment by crisbeto** [comment]
I don't think that we can change the arrows at this point, but we should be able to make it at least snap to the next tab.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
